src/reducers/entities/admin.js

```diff
@@ -221,6 +264,16 @@ function userAccessTokens(state = {}, action) {
 
         return {...state, [action.userId]: nextUserState};
     }
+    case AdminTypes.RECEIVED_USER_ACCESS_TOKENS: {
+        const nextUserState = {};
+
+        for (const uat of action.data) {
+            nextUserState[uat.user_id] = {...nextUserState[uat.user_id] || {}};
```

Comment:
Unless I'm missing something (and this isn't really a big deal), but I think it would be a little more efficient to do something like:

```javascript
if (!nextUserState[uat.user_id]) {
  nextUserState[uat.user_id] = {};
}
```
or if you don't want the if statement
```javascript
nextUserState[uat.user_id] = nextUserState[uat.user_id] || {};
```

I don't think it's necessary to make a new object every time with `...`

Reply:
I think its okay we do this. The example I can think of where this **may** fail would be this one:

```jsx
class Wrapper = (props) => {
    <InnerWrapper tokens={props.userAccessTokensByUser[props.userId]}>
}
```

However since we use the spread operator on `...nextUserState`, I think `<Wrapper>` will get re-rendered and therefore `<InnerWrapper>` will also get re-rendered.  So it should be okay. I made the change.